Feature/BookNow_TopMenu.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Book Now Booking Form from Top Menu Home Page
```

Comment:
create one feature file for the menu and test all the menu items

Reply:
Done